lms/djangoapps/grades/tasks.py

```diff
@@ -32,61 +32,37 @@ def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher):
         be updated only if the new grade is higher than the previous
         value.
     """
-    if not PersistentGradesEnabledFlag.feature_enabled(course_id):
+    course_key = CourseLocator.from_string(course_id)
+    if not PersistentGradesEnabledFlag.feature_enabled(course_key):
```

Comment:
So this confused me as well.  Since everywhere else in our code _(including tests)_ that calls feature_enabled` we pass a `CourseKey` object.  Only here we were passing a string object.  I made it consistent with all the other calls.  (It may have _just_ worked - since the `CourseKey` object eventually gets serialized anyway.)